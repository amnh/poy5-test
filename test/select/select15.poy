(* 'select' unique trees *)
set(seed:1,root:"t1")
read ("1.fas", "30.ss", aminoacids:("40.aa"))
build(6)
report(treestats)
select(unique)
report(treestats)
report(trees:(total))