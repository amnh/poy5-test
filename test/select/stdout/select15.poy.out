Trees Found:
                                            
              Tree length    Number of hits
              2787.          1
              2797.          1
              2804.          1
              2823.          1
              2824.          1
              2855.          1
Trees Found:
                                            
              Tree length    Number of hits
              2787.          1
              2797.          1
              2804.          1
              2823.          1
              2824.          1
              2855.          1
(t1,(t2,(((t3,t29),(t27,(t26,t25))),((t16,(t15,(t45,t35))),((t36,(t9,t8))
,(((t41,(t22,(t23,(t24,t21)))),(t43,((t44,t31),(t19,(t30,(t20,t18))))))
,(((t42,t17),(t10,(t39,((t48,t13),(t37,(t12,(t40,t14))))))),(t32,((t5,(t7
,t34)),(t4,(t46,(t49,(t33,(t28,(t6,(t11,(t47,t38)))))))))))))))))[2797.]
;
 (t1,(t2,(t37,((t16,(t15,(t45,t35))),((t29,t28),(t32,(t3,((((t9,t8),(t36,t10)
 ),(t19,(t41,(((t23,t22),(t24,t21)),((t30,(t44,t43)),(t31,(t42,(t39,t17))))))
 )),((t4,(t34,(t7,t5))),(((t48,t13),(t14,(t40,t12))),((t49,t46),(t33,(((t20
 ,t18),(t6,t11)),((t47,t38),(t27,(t26,t25))))))))))))))))[2824.];
                                                                 (t1,(t2,(t19
                                                                 ,(((t9,(t8
                                                                 ,(t36,t10)))
                                                                 ,(t32,(t3
                                                                 ,(t4,(t5,(t7
                                                                 ,t34))))))
                                                                 ,(((t15,(t16
                                                                 ,(t45,t35)))
                                                                 ,(t29,((t27
                                                                 ,(t26,t25))
                                                                 ,(t28,(t6
                                                                 ,t11)))))
                                                                 ,(t30,(((t20
                                                                 ,t18),(t41
                                                                 ,((t22,(t31
                                                                 ,(t44,t43)))
                                                                 ,(t23,(t24
                                                                 ,(t21,(t42
                                                                 ,t17)))))))
                                                                 ,(((t49,t46)
                                                                 ,(t33,(t40
                                                                 ,(t47,t38)))
                                                                 ),((t39,(t48
                                                                 ,t13)),(t14
                                                                 ,(t37,t12)))
                                                                 ))))))))
                                                                 [2804.]
                                                                 ;
                                                                  (t1,(t2
                                                                  ,((t29,t28)
                                                                  ,((t27,(t26
                                                                  ,t25))
                                                                  ,(((t4,(t34
                                                                  ,(t7,t5)))
                                                                  ,(t37,(t15
                                                                  ,(t16,(t45
                                                                  ,t35)))))
                                                                  ,((t9,(t8
                                                                  ,t36))
                                                                  ,((t32,t3)
                                                                  ,((t30,(t19
                                                                  ,((t20,t18)
                                                                  ,(t33,((t40
                                                                  ,(t14,t12))
                                                                  ,((t47,t38)
                                                                  ,(t6,t11)))
                                                                  )))),((t44
                                                                  ,t43),((t22
                                                                  ,(t41,(t23
                                                                  ,(t24,t21))
                                                                  )),(t31
                                                                  ,((t10,(t42
                                                                  ,t17))
                                                                  ,((t49,t46)
                                                                  ,(t39,(t48
                                                                  ,t13)))))))
                                                                  ))))))))
                                                                  [2855.]
                                                                  ;
                                                                   (t1,(t2
                                                                   ,((t29
                                                                   ,(t27,(t26
                                                                   ,t25)))
                                                                   ,(t3,((t34
                                                                   ,(t28,(t6
                                                                   ,t11)))
                                                                   ,((t14
                                                                   ,(t40,t12)
                                                                   ),(((t37
                                                                   ,(t39,(t48
                                                                   ,t13)))
                                                                   ,(t21
                                                                   ,((t10
                                                                   ,(t36,(t9
                                                                   ,t8)))
                                                                   ,(t16,(t15
                                                                   ,(t45,t35)
                                                                   )))))
                                                                   ,(((t49
                                                                   ,t46),(t4
                                                                   ,(t7,t5)))
                                                                   ,(t32
                                                                   ,((t30
                                                                   ,(t19
                                                                   ,((t20
                                                                   ,t18),(t33
                                                                   ,(t47,t38)
                                                                   )))),((t44
                                                                   ,t43),(t22
                                                                   ,(t24,(t23
                                                                   ,(t41,(t31
                                                                   ,(t42,t17)
                                                                   ))))))))))
                                                                   ))))))
                                                                   [2823.]
                                                                   ;
                                                                    (t1,(t2
                                                                    ,(t37
                                                                    ,((t15
                                                                    ,(t16
                                                                    ,(t45
                                                                    ,t35)))
                                                                    ,(((t31
                                                                    ,t23)
                                                                    ,(t24
                                                                    ,t21))
                                                                    ,((t22
                                                                    ,((t44
                                                                    ,t43)
                                                                    ,(t42
                                                                    ,t17)))
                                                                    ,(t41
                                                                    ,((t19
                                                                    ,((t20
                                                                    ,t18)
                                                                    ,(t33
                                                                    ,(((t47
                                                                    ,t38),(t6
                                                                    ,t11))
                                                                    ,(t40
                                                                    ,(t14
                                                                    ,(t12
                                                                    ,(t39
                                                                    ,(t48
                                                                    ,t13)))))
                                                                    )))),(t30
                                                                    ,(t32
                                                                    ,((t4
                                                                    ,((t49
                                                                    ,t46),(t5
                                                                    ,(t7,t34)
                                                                    ))),(((t8
                                                                    ,t36),(t9
                                                                    ,t10))
                                                                    ,(t3,(t29
                                                                    ,(t28
                                                                    ,(t27
                                                                    ,(t26
                                                                    ,t25)))))
                                                                    )))))))))
                                                                    )))
                                                                    [2787.]
                                                                    ;
                                                                    
                                                                    

